packages/web-components/src/components/va-textarea/va-textarea.tsx

```diff
@@ -154,22 +154,23 @@ export class VaTextarea {
   }
 
   render() {
-    const { 
-      label, 
-      error, 
-      placeholder, 
-      name, 
-      required, 
-      value, 
-      hint, 
-      uswds, 
-      charcount, 
-      messageAriaDescribedby 
+    const {
+      label,
+      error,
+      placeholder,
+      name,
+      required,
+      value,
+      hint,
+      uswds,
+      charcount,
+      messageAriaDescribedby
     } = this;
 
     const maxlength = this.getMaxlength();
-    const ariaDescribedbyIds = `${messageAriaDescribedby ? 'input-message' : ''} ${error ? 'error-message' : ''} 
-    ${charcount && maxlength ? 'charcount-message' : ''}`.trim() || null;
+    const ariaDescribedbyIds = `${error ? 'input-error-message' : ''} ${
+      charcount && maxlength ? 'charcount-message' : ''} ${
+      messageAriaDescribedby ? 'input-message' : ''}`.trim() || null;
```

Comment:
So the order of the IDs here are being updated because they are read in that specific order by a screen reader?

Reply:
I discussed this with @joshkimux, and he wasn't 100% sure, but it makes sense to keep the IDs in DOM-order, at least